### PR TITLE
add button variants to live modal preivew

### LINKED ISSUE #2118: [modal] live demo bugs
Live demo bugs for modal:
- [ ] Missing options to change footer button numbers (1,2,3 button)
- [x] Missing has scrolling content knob

### PR DESCRIPTION
**Pull request by dakahn**
Closes #2118 

Adds single, and triple button variant examples to the live preview on Code and usage page.

Double variant is the default so it wasn't added -- and no button is a knob on the right 👍🏽

### CODE REVIEW

File: src/pages/components/modal/code.mdx [hunk 1]
```diff
@@ -128,5 +136,56 @@ documentation, see the Storybooks for each framework below.
...
+          {
+            buttonText: 'Keep both',
+            onClick: action('onClick'),
+          },
+          {
+            buttonText: 'Rename',
+            onClick: action('onClick'),
+          }
+        ]
+      }
```
Comment (dakahn) [inline src/pages/components/modal/code.mdx:186]: @vpicone I can't seem to get a three button version to render correctly? AFAIK this should be good. Do you think you could have a look and see if there's something I'm missing as far as the live preview is concerned?

**Review by emyarod — COMMENTED** [review]
should this PR be converted to a draft in the meantime?

File: src/pages/components/modal/code.mdx (hunk 1 above)
Comment (dakahn) [reply src/pages/components/modal/code.mdx]: looks like it was our action prop getting called in a way Storybook didn't like. Thanks @jnm2377

**Review by aagonzales — APPROVED** [review]
Looks good to me!

**Review by jnm2377 — APPROVED** [review]

**Review by jillianhowarth — APPROVED** [review]
LGTM!

**Pull request merged**